src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -345,20 +348,22 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
             lowContrast
           />
         )}
-        {this.state.submitError && (
+        {this.state.deleteError && (
```

Comment:
Not for this PR but just spotted that were missing error handling for cancelling a run :see_no_evil:

https://github.com/tektoncd/dashboard/blob/266f8030f92a3d321ba648b1d97512f8c2280dfe/src/containers/PipelineRuns/PipelineRuns.js#L115

Reply:
so we are... created an issue to track that: https://github.com/tektoncd/dashboard/issues/1884